9control.tex

```diff
@@ -1189,8 +1189,8 @@ \subsubsection{Recruitment Deviation Setup}
 
 	1 \Tstrut & Do Recruitment Deviations & This selects the way in which recruitment deviations are coded: \\
 	  & & 0: None (so all recruitments come from spawner recruitment curve). \\
-	  & & 1: Deviation vector (previously the only option): the deviations are encoded as a deviation vector, so ADMB enforces a sum-to-zero constraint. \\
-	  & & 2: Simple deviations: the deviations do not have an explicit constraint to sum to zero, although they still should end up having close to a zero sum. The difference in model performance between options (1) and (2) has not been fully explored to date. \\
+	  & & 1: Deviation vector (previously the only option): the deviations are encoded as a deviation vector, so ADMB enforces a sum-to-zero constraint, but only in the main period. \\
```

Comment:
revise to say "the deviations during the main period are encoded as a deviation vector that enforces them to sum to zero for this period." or something along those lines.